### DISCUSSION TITLE
Add Feishu Oauth client

### DISCUSSION THREAD

**Discussion by CHEN-Jing194** [frankie567/httpx-oauth#315] [General]
Hello,

Thank you very much for your outstanding project. I have successfully implemented the Feishu OAuth2 process using your project. Would it be possible for me to contribute the Feishu OAuth client I developed to your project? Feishu (also known as Lark outside of China) is a popular team collaboration platform in China, and I believe this would be beneficial to many Chinese developers. Once again, thank you for your exceptional work.

Best regards,

**Comment by frankie567** [comment]
Why not! We already have some local services implemented, so yes, open to review a PR with it!
